modulefiles/clas12/4.5

```diff
@@ -20,9 +20,9 @@ prereq --optional root/6.30.04
 prereq --optional ccdb/1.99.1
 prereq --optional rcdb/1.99.0
 prereq --optional qadb/1.3.0
-prereq --optional hipo/4.0.1
+prereq --optional hipo/4.1.0
```

Comment:
Should we switch the `hipo` version now? AFAIK there aren't any breaking changes.

Reply:
we can only change the clas12/dev module until gemc is ready for 4.1.0